components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -0,0 +1,258 @@
+
+  const setDate = dateRange => {
+    if (!dateRange) {
+      setDateFrom(null);
+      setDateTo(null);
+      refetch();
+      return;
+    }
+    const dates = dateRange.split('→');
+    const dateFrom = dates[0];
+    const dateTo = dates[1];
+    if (dateFrom === 'all') {
+      setDateFrom(null);
+    } else {
+      setDateFrom(dateFrom);
+    }
+    if (dateTo === 'all') {
+      setDateTo(null);
+    } else {
+      setDateTo(dateTo);
+    }
+  };
+
+  const getDateString = () => {
+    let dateString;
+    if (!dateFrom && !dateTo) {
+      return;
+    }
+    if (dateFrom) {
+      dateString = `${dateFrom}→`;
+    } else {
+      dateString = 'all→';
+    }
+
+    if (dateTo) {
+      dateString += dateTo;
+    } else {
+      dateString += 'all';
+    }
+    return dateString;
+  };
+
+  const setCollectiveFilter = collectives => {
+    if (collectives.length === 0) {
+      setCollectives(null);
+    } else {
+      const collectiveIds = collectives.map(collective => ({ legacyId: collective.value.id }));
+      setCollectives(collectiveIds);
+    }
+  };
+
+  return (
+    <React.Fragment>
+      <Flex flexWrap="wrap">
+        <Container width={[1, 1, 1 / 2]} pr={2} mb={[3, 3, 0, 0]}>
+          <FilterLabel htmlFor="transactions-period-filter">
+            <FormattedMessage id="TransactionsOverviewSection.PeriodFilter" defaultMessage="Filter by Date" />
+          </FilterLabel>
+          <PeriodFilter onChange={value => setDate(value)} value={getDateString()} />
+        </Container>
+        <Container width={[1, 1, 1 / 2]}>
+          <FilterLabel htmlFor="transactions-collective-filter">
+            <FormattedMessage id="TransactionsOverviewSection.CollectiveFilter" defaultMessage="Filter by Collective" />
+          </FilterLabel>
+          <CollectivePickerAsync
+            inputId="TransactionsCollectiveFilter"
+            data-cy="transactions-collective-filter"
+            types={['COLLECTIVE']}
+            isMulti
+            onChange={value => setCollectiveFilter(value)}
+          />
+        </Container>
+      </Flex>
+      {loading ? (
+        <Loading />
+      ) : (
+        <Flex flexWrap="wrap" mt={18} mb={12}>
+          <FundAmounts>
+            <TotalFundsLabel minWidth="280px">
+              <P>
+                <Span fontWeight="500">
+                  {numContributions} <FormattedMessage id="Contributions" defaultMessage="Contributions" />
+                </Span>{' '}
+                | <FormattedMessage id="DailyAverage" defaultMessage="Daily avg" />
+                {': '}
+                <Span fontWeight="700">{formatCurrency(dailyAvgIncome, currency)}</Span>
+              </P>
+            </TotalFundsLabel>
+            <TotalFundsLabel
+              minWidth="250px"
+              position="relative"
+              left={['-280px', '-280px', '-280px', '100px']}
+              top={['85px', '85px', '85px', '0px']}
```

Comment:
Not a responsive approach, as it depends on the parent container size. Let's use CSS's flex or similar mechanism.

Could be worth extracting in a separate component since we have the use case multiple times on the page.

Reply:
I'll open a separate PR for this soon using the flex approach. 👍🏽

Reply:
Just for the record this will be done at https://github.com/opencollective/opencollective/issues/4690